package.json

```diff
@@ -21,11 +20,12 @@
   "scripts": {
     "lint": "tslint -p ./tsconfig.json",
     "start": "react-scripts start",
-    "clean": "rm -rf build",
-    "build": "GENERATE_SOURCEMAP=true react-scripts build",
+    "start:static": "npm run-script build && NODE_ENV=${NODE_ENV:-test} npm run-script prerender && npm run-script server",
```

Comment:
doest the NODE_ENV default get applied to the `npm run-script server` as well? it needs to be for the proxies to work properly

Reply:
oh, it does not. What if `"server": ...` also contains the `NODE_ENV=${NODE_ENV:-test}` default?

Reply:
ok, how about this (see commit below)?